react/tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -1,42 +1,56 @@
 import React, { Component } from 'react';
 import Spinner from 'react-spinkit';
+import { connect } from 'react-redux';
+import { arrayOf, func, bool } from 'prop-types';
 
-import getMatches from '../../../services/MatchesService';
+import { matchesPropsTypes } from '../../../constants/propsTypes';
+import dataActions from '../../../redux/data/actions';
 
 import Matches from './components/Matches';
 import styles from './styles.module.scss';
 import Board from './components/Board';
 
 class Game extends Component {
-  state = {
-    matches: [],
-    loading: true
-  };
-
   componentDidMount() {
-    setTimeout(() => getMatches.getMatches().then(response => {
-      this.getData(response.data);
-    }), 500);
+    const { handleGetData } = this.props;
+    setTimeout(() => handleGetData(), 500);
   }
 
-  getData = (matches) => this.setState({ matches, loading: false })
-
   render() {
+    const { data } = this.props;
     return (
       <div className={styles.game}>
         <div className={styles.gameBoard}>
           <Board />
         </div>
         <div className={styles.gameInfo}>
           {
-            this.state.loading
+            this.props.loading
               ? <Spinner name="circle" fadeIn="none" />
-              : <ol className={styles.infoContainer}>{this.state.matches.map(Matches)}</ol>
+              : <ol className={styles.infoContainer}>{ data.map(Matches) }</ol>
           }
         </div>
       </div>
     );
   }
 }
 
-export default Game;
+Game.propTypes = {
+  data: arrayOf(matchesPropsTypes).isRequired,
+  handleGetData: func.isRequired,
+  loading: bool
+};
+
+const mapDispatchToProps = dispatch => ({
+  handleGetData: () => dispatch(dataActions.getData())
+});
+
+const mapStateToProps = state => ({
+  data: state.data.data,
```

Comment:
i'd rename the reducer to Game, so here it'd be `state.game.data`, so its more representative of what it is